src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const thumbsList = document.getElementById('thumbs');
+const largeImg = document.getElementById('largeImg');
+
+thumbsList.addEventListener('click', (clickEvent) => {
+  const imageUrl = clickEvent.target.closest('a').href;
```

Comment:
This line might cause an error if the clicked element doesn't have a parent 'a' element. It's better to check if the parent 'a' element exists before accessing its href.